packages/codegen-ui-react/lib/react-studio-template-renderer.ts

```diff
@@ -1296,5 +1302,31 @@ export abstract class ReactStudioTemplateRenderer extends StudioTemplateRenderer
     return undefined;
   }
 
+  /* To map predefined props to children props for variants
+   *
+   * Example: Text prop label is mapped to to Text prop Children
+   *
+   */
+  private mapSyntheticPropsForVariants() {
+    if (!isStudioComponentWithVariants(this.component)) {
+      return;
+    }
+
+    this.component.variants.forEach((variant) => {
+      // loop through the keys in the dict. Ex. button, button.text
+      Object.keys(variant.overrides).forEach((overridesKey) => {
```

Comment:
Instead of `Object.key` you can use `Object.entries` since you are accessing the values as well as the keys down below.